src/main/java/uk/ac/ebi/eva/pipeline/parameters/InputParameters.java

```diff
@@ -63,7 +63,6 @@
     @Value(PARAMETER + JobParametersNames.CONFIG_CHUNK_SIZE + "']?:1000}")
     private int chunkSize;
```

Comment:
Yes, it can be done in another pull request. I'm not sure about some details, like should we leave them as job parameter injection ("#{jobParameters['config.chunk.size']}") or direct injection ("${config.chunk.size}"). Maybe we would want to change the chunk size depending on the job, but it only will affect the populationStatistics step, because the other ones are built before the jobparameters are resolved, so a change in a job parameter won't affect the job instantiation